src/utility/getRuntimeEnv.ts

```diff
@@ -0,0 +1,68 @@
+export default () => {
+  const isContentOneDomain =
+    window.location.hostname.includes(".cms.content.one");
+
+  if (isContentOneDomain) {
+    const COOKIE_DOMAIN = ".content.one";
+
+    switch (process.env.NODE_ENV) {
+      case "production":
+        return {
+          URL_MANAGER: ".cms.content.one",
+          COOKIE_DOMAIN,
+        };
+
+      case "stage":
+        return {
+          URL_MANAGER: ".stage.cms.content.one",
```

Comment:
The domain structure should be `cms.ENV.content.one`. Wanting that to remain consistent with the zesty.io domain